assets/js/aplus.js

```diff
@@ -666,27 +670,30 @@ $(function() {
  * Note! Translation requires that at least 'hl' query parameter is copied
  */
 (function(window) {
-    window.addEventListener('click', function(event) {
+
+  $(document).on("click", "a[href]", function(event){
         if (
-            event.target.nodeName === 'A'
-            && event.target.href
-            && event.target.protocol === window.location.protocol
+            event.target.protocol === window.location.protocol
             && event.target.host === window.location.host // hostname:port
             && (event.target.search === '' || event.target.search === '?') // no search
         ) {
-            event.preventDefault(); // stop default redirect
+            //event.preventDefault(); // stop default redirect
+            const searhLanguage = new URLSearchParams(window.location.search);
+            const lang = searhLanguage.get('hl');
             event.target.search = window.location.search; // copy query params
-            if (event.target.getAttribute('target') === '_blank') {
-                // Open a new window or tab.
-                window.open(
-                    event.target.href,
-                    '_blank',
-                    'location,menubar,resizable,scrollbars,status,noopener');
-            } else {
-                window.location.href = event.target.href; // redirect the current window
-            }
         }
-    }, false);
+    });
+
+    if (!window.location.search) {
+      if(document.documentElement.lang == "en-gb" || document.documentElement.lang == "en")
```

Comment:
Is it necessary to check the documentElement's lang attribute here? The original idea was to copy to the hl query parameter only if the current location (URL) contains the hl parameter. It can be copied from the location, thus it doesn't matter what values the document lang attribute has.

The issue with the document lang attribute can be fixed (https://github.com/apluslms/a-plus/issues/748), but I don't think it is necessary in this PR.